extensions/cornerstone-dicom-sr/src/tools/DICOMSRDisplayTool.ts

```diff
@@ -270,11 +274,38 @@ export default class DICOMSRDisplayTool extends AnnotationTool {
     viewport,
     renderableData,
     annotationUID,
+    referencedImageId,
     options
   ) {
-    let canvasCoordinates;
+    const canvasCoordinates = [];
     renderableData.map((data, index) => {
-      canvasCoordinates = data.map(p => viewport.worldToCanvas(p));
+      const point = data[0];
+      // This gives us one point for arrow
+      canvasCoordinates.push(viewport.worldToCanvas(point));
```

Comment:
I get why we are doing all of this, but it does seem a bit weird. What about when we start storing probe. I guess we render them both as arrows? Is there any harm? cc @swederik